web/auth.ex

```diff
@@ -20,7 +20,11 @@ defmodule HexWeb.Auth do
     case result do
       {user, key} ->
         if Comeonin.Tools.secure_check(key.secret_second, second) do
-          {:ok, user}
+          if key.revoked_at === nil do
```

Comment:
Whoops, it looks like I've been doing `=:=` in Erlang and `===` in Elixir for a while now without realizing that it only has an effect between integers and floats.  I'll change that out.